spinalcordtoolbox/deepseg/models.py

```diff
@@ -360,6 +340,21 @@
                              'dataset has healthy control subjects. The model has been trained in a human-in-the-loop active learning fashion.',
          'url': 'https://github.com/sct-pipeline/fmri-segmentation',
          'models': ['model_seg_sc_epi_nnunet']},
+    'seg_ms_lesion_mp2rage':
+        {'description': 'Segmentation of spinal cord MS lesions on MP2RAGE UNIT1 contrast',
```

Comment:
The cropping requirement is only mentioned in `long_description`, which is only shown when users type `sct_deepseg -list-tasks`. 

It might be good to also mention "cropped" in the regular description, too. We could keep the old description, maybe?

```suggestion
    'seg_ms_lesion_mp2rage':
        {'description': 'MS lesion segmentation on cropped MP2RAGE data',
```

Reply:
Note that we also want to keep the description short, so that everything fits in the table when we type `sct_deepseg -h`.